scripts/maven/maven_dependencies_list.kts

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env kscript
```

Comment:
See my doc comment: I think it's preferable to change this to be an actual Kotlin binary file (i.e. with a main() function & a Bazel definition) so that it can be run through Bazel.

However, that may pose a slight problem since this script is also executing Bazel commands. Can you maybe see if running this script with bazel run & it, in turn, using bazel query works, or if it hangs? If it hangs, you may need to set the ``output_base`` of the bazel query command to be something like ``/tmp`` / different from the script so that they run separately.

Reply:
I tried setting up the output_base to `/tmp` and `/home/prayutsu/c1` (random) but it still isn't getting executed properly.
The full command is - 

`bazel --output_base=/tmp query 'deps(deps(//:oppia) intersect //third_party/...) intersect @maven//...'`

When run on terminal directly, both got executed successfully.
It is not hanging, the process is returning a non-zero status code and thus the program is getting terminated.

Reply:
See my other comment--try using the same BazelClient approach as #3374. I found that the output_base wasn't actually needed.